<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Consulta Datos Procesamiento Industrial UEES</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" rowspan="3" colspan="4">
     <widget class="QLabel" name="label">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../imagenes.qrc">:/img/LogoUEES.png</pixmap>
      </property>
     </widget>
    </item>
    <item row="0" column="4" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Ink Free</family>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(133, 33, 62)</string>
      </property>
      <property name="text">
       <string>Grupo 3</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="4" colspan="3">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Ink Free</family>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(133, 33, 62)</string>
      </property>
      <property name="text">
       <string>Lenguajes de Programaci√≥n</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4" colspan="3">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Ink Free</family>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(133, 33, 62)</string>
      </property>
      <property name="text">
       <string>Procesamiento de Datos JSON</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="7">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Tipos de busqueda</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="3" colspan="4">
        <widget class="QLineEdit" name="lineEditBuscar"/>
       </item>
       <item row="0" column="5" colspan="2">
        <widget class="QDateEdit" name="dateEditFin">
         <property name="displayFormat">
          <string>dd/MM/yyyy</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="6">
        <widget class="QRadioButton" name="radioButtonApellido">
         <property name="text">
          <string>Apellido</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>5</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" rowspan="4">
        <widget class="QPushButton" name="pushButtonAll">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>80</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 127, 191); color: rgb(255, 255, 255)</string>
         </property>
         <property name="text">
          <string>Todos</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QRadioButton" name="radioButtonId">
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButtonRange">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 170, 255); color: rgb(255, 255, 255)</string>
         </property>
         <property name="text">
          <string>Rango Fecha</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4" colspan="2">
        <widget class="QRadioButton" name="radioButtonNombre">
         <property name="text">
          <string>Nombre</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="pushButtonPerson">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 170, 255); color: rgb(255, 255, 255)</string>
         </property>
         <property name="text">
          <string>Datos Persona</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3" colspan="2">
        <widget class="QDateEdit" name="dateEditInicio">
         <property name="displayFormat">
          <string>dd/MM/yyyy</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="0" colspan="7">
     <widget class="QTableWidget" name="tableWidget">
      <column>
       <property name="text">
        <string>Id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Fecha</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nombre</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Apellido</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>KlosP</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="5" column="0">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>60</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="labelTiempo">
      <property name="text">
       <string>Tiempo: 0 ms</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="3">
     <widget class="QLabel" name="labelRegistros">
      <property name="text">
       <string>Registros: 0</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="5">
     <widget class="QLabel" name="labelPeso">
      <property name="text">
       <string>Total peso: 0</string>
      </property>
     </widget>
    </item>
    <item row="5" column="6">
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>135</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <tabstops>
  <tabstop>pushButtonAll</tabstop>
  <tabstop>pushButtonRange</tabstop>
  <tabstop>dateEditInicio</tabstop>
  <tabstop>dateEditFin</tabstop>
  <tabstop>pushButtonPerson</tabstop>
  <tabstop>radioButtonId</tabstop>
  <tabstop>radioButtonNombre</tabstop>
  <tabstop>radioButtonApellido</tabstop>
  <tabstop>lineEditBuscar</tabstop>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources>
  <include location="../imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
